packages/core/src/components/select/select.tsx

```diff
@@ -67,6 +136,8 @@ export class AtomSelect {
   }
 
   private handleBlur = () => {
+    this.setTag()
```

Comment:
is a nice idea to add a short circuit to avoid to start it when tags are not active

```suggestion
    this.tag && this.setTag()
```

Reply:
a little different but same idea, done here
https://github.com/juntossomosmais/atomium/pull/549/commits/364ccc3ae96b973a94d3c4b22518e392f9986055